metricbeat/module/php_fpm/pool/_meta/fields.yml

```diff
@@ -46,8 +59,30 @@
         description: >
           The number of servers current processing a page - the minimum is `1`
           (so even on a fully idle server, the result will be not read `0`).
+      - name: total
+        type: long
+        description: >
+          The number of idle + active processes.
+      - name: max_active
+        type: long
+        description: >
+          The maximum number of active processes since FPM has started.
+      - name: max_children_reached
+        type: long
+        description: >
+          Number of times, the process limit has been reached,
+          when pm tries to start more children (works only for
+          pm 'dynamic' and 'ondemand').
     - name: slow_requests
       type: long
       description: >
         The number of times a request execution time has exceeded
         `request_slowlog_timeout`.
+    - name: start_since
+      type: long
+      description: >
+        Number of seconds since FPM has started.
+    - name: start_time
+      type: long
```

Comment:
I think this could be `date` https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html, you may need to convert the value before pushing it to Elasticsearch

Reply:
Sure, it needs a `epoch_second` format, let me see if I can find how to format that